### PR TITLE
Add osu (2015 October 17)

### PR DESCRIPTION
**Pull request by numbermaniac**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/osu.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'osu' do
+  version '20151017'
+  sha256 '5677ee299ea9e271546e6ab56b9d3abe002d4295c9529a5b917f5af167dcd477'
+
+  url 'http://m1.ppy.sh/osu%21-20151017.dmg'
```
Comment (vitorgalvao) [inline Casks/osu.rb]: `url "http://m1.ppy.sh/osu%21-#{version}.dmg"`

**Comment by Numbermaniac** [comment]
Done, I think. Not sure why it says the checks have failed...

File: Casks/osu.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'osu' do
+  version '20151017'
+  sha256 '5677ee299ea9e271546e6ab56b9d3abe002d4295c9529a5b917f5af167dcd477'
+
+  url 'http://m1.ppy.sh/osu%21-#{version}.dmg'
```
Comment (vitorgalvao) [inline Casks/osu.rb]: You need to use double quotes here, for it to interpret the interpolation correctly. That’s why it is failing.

**Comment by Numbermaniac** [comment]
Ah, I see. Done.

**Pull request merged**